src/ekat/kokkos/ekat_kokkos_utils.hpp

```diff
@@ -433,10 +435,22 @@ template <typename ValueType, typename ExeSpace = Kokkos::DefaultExecutionSpace>
 class TeamUtils : public TeamUtilsCommonBase<ValueType, ExeSpace>
 {
  public:
+  TeamUtils() = default;
+
   template <typename TeamPolicy>
   TeamUtils(const TeamPolicy& policy, const double& = 1.0) :
     TeamUtilsCommonBase<ValueType, ExeSpace>(policy)
   { }
+
+  TeamUtils& operator= (const TeamUtils& src)
```

Comment:
Isn't this the default implementation? I assume the base class operator= is invoked in the default?

Reply:
I agree, if you need a "default" behavior, you should simply declare `operator=(..) = default`. The defaulted impl, implicitly calls the base class op=.